### ISSUE TITLE
continue pretraining nomic

### ISSUE THREAD

**Issue by keyuchen21** [nomic-ai/contrastors#70]
1. any guide/code on how can I do continue pre-training on nomic v1.5 to align with a specific domain area? like finance or medicine.
2. will just do Masked Language Modeling Pretraining on medical data works in this case? (based on this paper's idea: https://arxiv.org/pdf/2004.10964) 
- since Nomic embed have additional Unsupervised and Supervised Contrastive Pretraining, I'm not sure if I can still do additional masked langauge modeling pre-training on a domain data........

Best,
Keyu
